README.md

```diff
@@ -83,9 +83,10 @@ Device         Boot   Start      End  Sectors  Size Id Type
 - `mmcblkXp7` is the writable `etc` overlay partition (ext4 filesystem mounted as overlayfs on `/etc`)
 - `mmcblkXp8` is the writable `data` partition with ext4 filesystem
 
-**Note1**: The size of `data` depends on your sdcard/emmc/nvme size. Per default it has a size of 512M and is resized on the first boot to the max available size.<br>
-**Note2**: The size of `rootA` and `rootB` depends on your board and image variant (development or release).<br>
-**Note3 (only for images where u-boot is used as OS bootloader)**: There is a reserved area between the boot partition and the rootA partition used for two redundant u-boot environment banks. For this purpose, the following configuration variables are used:
+**Note1**: The partition layout is just an example and may differ in a series product.<br>

```

Comment:
-> "The partition layout is an example, sizes may differ depending on your board and image variant (development or release)."
\+ delete note 3?

Reply:
fine for me, done